tsdb/testutil.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2017 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tsdb
+
+import (
+	"testing"
+
+	prom_testutil "github.com/prometheus/client_golang/prometheus/testutil"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/prometheus/prometheus/model/histogram"
+	"github.com/prometheus/prometheus/model/labels"
+	"github.com/prometheus/prometheus/storage"
+	"github.com/prometheus/prometheus/tsdb/chunks"
+)
+
+const (
+	float = "float"
+)
+
+type tsValue struct {
```

Comment:
I've tried to move this testutil.go file into tsdbutil directory but it's a big refactor. It's using unexported stuff from tsdb which would need moving/renaming and also there's now non test related stuff in there (dir locker). So it's a bit of a mess.

So if we're keeping this here , I'd suggest:
```suggestion
type testValue struct {
```
to make clear the intention. VS code should be able to just rename it.